### PR TITLE
Missing vote list and delegate username in the transaction details page - Closes #419

### LINKED ISSUE #419: Missing vote list and delegate username in the transaction details page.
### Expected behavior
We should be able to see all the important data of transactions in the transactions details page.

### Actual behavior
 - Delegate registration: the username is missing.
 - Vote: votes list is missing.

### PR DESCRIPTION
**Pull request by colado**
# What was the bug or feature?
Described in #419.

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
Check transactions `Vote` and `Delegate registration`.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by colado** [comment]
@reyraa I've added the loader and the checks for active token as we discussed. Let me know if anything else needs to be changed.

File: locales/resources/de.json [hunk 1]
```diff
@@ -120,6 +125,7 @@
   "Transfer your LSK tokens easily to other accounts by simply scanning their QR code.": "Einfach den QR code scannen, um LSK Tokens an ein anderes Konto zu verschicken.",
   "Unauthorized! Please try again.": "Nicht autorisiert! Nochmal versuchen.",
   "Verification": "Bestätigen",
+  "Voter": "",
```
Comment (reyraa) [inline locales/resources/de.json]: Wähler(in)

File: locales/resources/de.json [hunk 2]
```diff
@@ -82,6 +85,8 @@
   "Recipient": "Empfänger",
   "Reference": "Nachricht",
   "Reference (optional)": "Nachricht (optional)",
+  "Registrant": "",
+  "Removed votes": "",
```
Comment (reyraa) [inline locales/resources/de.json]: Entfernte Stimmen

File: locales/resources/de.json [hunk 3]
```diff
@@ -38,6 +40,7 @@
   "Copy to clipboard": "In Zwischenablage kopieren",
   "Copyright": "Copyright",
   "Couldn’t recognize the address or label. Please make sure it’s correct.": "Die Adresse oder Bezeichnung konnte nicht erkannt werden. Bitte überprüfe sie.",
+  "Delegate username": "",
```
Comment (reyraa) [inline locales/resources/de.json]: Spitzname des Delegierter

File: locales/resources/de.json [hunk 4]
```diff
@@ -13,10 +13,12 @@
   "Account address": "Kontoadresse",
   "Account creation": "Kontoerstellung",
   "Account initialization": "Kontoinitialisierung",
+  "Account nickname": "",
   "Activity": "Aktivitäten",
   "Add a reference": "Nachricht hinzufügen",
   "Add a reference to this transaction.": "Eine Nachricht dieser Transaktion hinzufügen.",
   "Add to bookmarks": "Zu Lesezeichen hinzufügen",
+  "Added votes": "",
```
Comment (reyraa) [inline locales/resources/de.json]: Hinzugefügte Stimmen

File: locales/resources/de.json [hunk 5]
```diff
@@ -13,10 +13,12 @@
   "Account address": "Kontoadresse",
   "Account creation": "Kontoerstellung",
   "Account initialization": "Kontoinitialisierung",
+  "Account nickname": "",
```
Comment (reyraa) [inline locales/resources/de.json]: Where is this used?

File: locales/resources/de.json [hunk 6]
```diff
@@ -82,6 +85,8 @@
   "Recipient": "Empfänger",
   "Reference": "Nachricht",
   "Reference (optional)": "Nachricht (optional)",
+  "Registrant": "",
```
Comment (reyraa) [inline locales/resources/de.json]: Registrant(in)

**Pull request merged**